backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc

```diff
@@ -77,8 +77,9 @@ spec:
 <2> Optional: Specify an array of resources to include in the backup. Resources might be shortcuts (for example, 'po' for 'pods') or fully-qualified. If unspecified, all resources are included.
 <3> Optional: Specify an array of resources to exclude from the backup. Resources might be shortcuts (for example, 'po' for 'pods') or fully-qualified.
 <4> Specify the name of the `backupStorageLocations` CR.
-<5> Map of {key,value} pairs of backup resources that have *all* the specified labels.
-<6> Map of {key,value} pairs of backup resources that have *one or more* of the specified labels.
+<5> The `ttl` field defines the retention time of the created backup and the backed up data. For example, if you are using Restic as the backup tool, the backed up data items and data contents of the persistent volumes (PVs) is stored until the backup expires. But storing this data consumes more space in the target backup locations. An additional storage will be consumed with frequent backups, which are created even before other in-progress backups might have timed out.
```

Comment:
"even before other in-progress backups might have timed out."  using the term "in-progress" for a backup that has completed but has not yet expired it's TTL is confusing.  "in-progess" backups typically refer to a backup that is currently executing the actual backup.

Unless someone has a better way to describe this state, I would say something like "unexpired completed backups"

This comment holds true for the other sections of the doc as well where "in-progress" is used to describe completed backups that have not expired their ttl. ( time to live )

Reply:
thanks, I've added that